moviepal.py

```diff
@@ -1,70 +1,102 @@
 #!/usr/bin/python3
-from bs4 import BeautifulSoup as bs
-from requests import get
-from re import findall
 import re
-import string
-from json import loads, load
-from time import sleep
 import webbrowser
+import json
+from re import findall
+from time import sleep
 
+from bs4 import BeautifulSoup as bs
+from requests import get
+import pandas as pd
 
-class mp():
 
+class EarningsCell():
+    def __init__(self, match):
+        self.daily_gross = match['daily_gross']
+        self.percent_change = match['percent_change']
+        self.average = match['average']
+        self.total_gross = match['total_gross']
+        self.day_number = match['day_number']
+
+    def __str__(self):
+        return f'Daily:{self.daily_gross} Change:{self.percent_change} Average:{self.average} Total:{self.total_gross} Day:{self.day_number}'
+
+class mp():
     url = 'http://www.omdbapi.com/'
     parameters = {'apikey': ''}
 
+    # TODO setting the pandas options here is really gross.
+    # TODO consider not printing right from the dataframe in boxoffice (or elsewhere)
+    pd.set_option('display.height', 1000)
```

Comment:
I'm receiving an error pertaining to this. Not sure how to solve or what's going on, really.
``` 
File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pandas/core/config.py", line 83, in _get_single_key
    raise OptionError('No such keys(s): {pat!r}'.format(pat=pat))
pandas.core.config.OptionError: "No such keys(s): 'display.height'"
```